new-lamassu-admin/src/pages/Wizard/components/Wallet/ChooseWallet.js

```diff
@@ -152,6 +152,21 @@ const ChooseWallet = ({ data: currentData, addData }) => {
           />
         </>
       )}
+      {selected === 'coinbasepro' && (
+        <>
+          <H4 noMargin>Enter wallet information</H4>
+          <FormRenderer
+            value={accounts.coinbasepro}
+            save={saveWallet(selected)}
+            elements={schema.coinbasepro.elements}
+            validationSchema={schema.coinbasepro.getValidationSchema(
+              accounts.coinbasepro
+            )}
+            buttonLabel={'Continue'}
+            buttonClass={classes.formButton}
+          />
+        </>
+      )}
```

Comment:
We need to start generalizing this code. Right now, `bitgo`'s schema takes a coin as parameter, but we could add the coin parameter to every wallet schema to make generalization easier. With the services revamp PR, my suggestion will vary a bit for sure, but I'd go something like this

```suggestion
      {selected && (
        <>
          <H4 noMargin>Enter wallet information</H4>
          <FormRenderer
            value={accounts[selected]}
            save={saveWallet(selected)}
            elements={schema[selected](coin).elements}
            validationSchema={schema[selected](coin).getValidationSchema(
              accounts.[selected]
            )}
            buttonLabel={'Continue'}
            buttonClass={classes.formButton}
          />
        </>
      )}
```

Reply:
I'm leaving this as it is and we will deal with the conflicts afterwards.